lib/ember_cli/shell.rb

```diff
@@ -59,13 +78,15 @@ def spawn(command)
       ) || exit(1)
     end
 
-    def exec(command)
+    delegate :run, :run!, to: :runner
```

Comment:
What do you think about moving this line to the top of the `private` method declarations, near the `attr_accessor` and `attr_reader` lines?